autotest/config.yaml

```diff
@@ -191,10 +194,43 @@ quatization_case_config:
         - deepseek-ai/deepseek-vl-1.3b-chat
         - deepseek-ai/deepseek-coder-1.3b-instruct
         - codellama/CodeLlama-7b-Instruct-hf
+        - THUDM/glm-4-9b-chat
+
+pytorch_quatization:
+    4bits:
+        - meta-llama/Meta-Llama-3-8B-Instruct
+        - meta-llama/Llama-2-7b-chat-hf
+        - internlm/internlm2_5-7b-chat
+        - internlm/internlm2_5-7b-chat-1m
+        - internlm/internlm2-chat-7b
+        - internlm/internlm2-chat-20b
+        - baichuan-inc/Baichuan2-7B-Chat
+        - 01-ai/Yi-6B-Chat
+        - Qwen/Qwen2-7B-Instruct
+        - Qwen/Qwen2-1.5B-Instruct
+        - Qwen/Qwen1.5-7B-Chat
+        - Qwen/Qwen-7B-Chat
     w8a8:
         - meta-llama/Meta-Llama-3-8B-Instruct
         - meta-llama/Llama-2-7b-chat-hf
         - internlm/internlm2-chat-20b
         - internlm/internlm2-chat-7b
         - 01-ai/Yi-6B-Chat
         - internlm/internlm2-20b
+
+
+longtext_model:
+    - meta-llama/Meta-Llama-3-1-8B-Instruct
+    - meta-llama/Meta-Llama-3-8B-Instruct
+    - meta-llama/Meta-Llama-3-1-70B-Instruct
+    - internlm/internlm2_5-7b-chat
```

Comment:
why not internlm2_5-7b-chat-1m?

Reply:
> why not internlm2_5-7b-chat-1m?

I add internlm2_5-7b-chat-1m into longtext generation benchmark. Still use internlm2_5-7b-chat in other benchmark test.